File: 백준/Bronze/1546. 평균/평균.py
## 최댓값 M, 과목의 수 N
## 점수/M * 100의 평균 -> 반환값은 float 타입

n = int(input())
arr = list(map(float,input().split(" ")))
print(sum(arr)*100/max(arr)/n)

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;
import java.util.Arrays;

public class Main{
    public static void main(String[] arge){
        Scanner sc =  new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        
        for (int i = 0; i < n; i++) arr[i] = sc.nextInt();
        // stream 으로 변환하여 내장 메서드 이용
        System.out.print(Arrays.stream(arr).sum() * 100.0 / Arrays.stream(arr).max().getAsInt() / n);
    }
}